base/show.jl

```diff
@@ -1992,7 +2007,7 @@ function showarray(io::IO, X::AbstractArray, repr::Bool = true; header = true)
         # override usual show method for Vector{Method}: don't abbreviate long lists
         io = IOContext(io, :limit => false)
     end
-    (!repr && header) && print(io, summary(X))
```

Comment:
This change seems unrelated?

Reply:
Right, but probably not worth a PR?

Reply:
Oh, and it's not completely unrelated since it's needed so that the module context is passed to `summary` via `io`. So it's useful on its own, but needed for this PR.